src/content/ui/navigation/index.md

```diff
@@ -89,12 +84,10 @@ To use the Router, switch to the `router` constructor on `MaterialApp` or
 such as [go_router][], typically provide a
 configuration for you. For example:
 
+<?code-excerpt "ui/navigation/lib/navigator_router.dart (push-route)"?>
```

Comment:
Same here. It's really nice to have a navigation example as well now, but the introduction text ("typically provide a
configuration for you") doesn't match anymore as the `MaterialApp.router` construction/configuration example is lost.

I feel it'd be nice to keep both, but if you think the declaration is out of the scope of this page, feel free to instead adjust the preceding text :)

Reply:
See note above.